bridge/src/commonMain/kotlin/ru/herobrine1st/matrix/bridge/telegram/TelegramWorker.kt

```diff
@@ -196,6 +200,60 @@ class TelegramWorker(
                                         RoomMessageEventContent.TextBased.Text(body = text),
                                     )
                                 }
+
+                                message.photo?.maxByOrNull { it.fileSize ?: 0 }?.let { photo ->
+
+                                    val sender = message.from!!
+
+                                    val fileInfo = withContext(Dispatchers.IO) {
+                                        bot.getFile(photo.fileId).toResult().get()
+                                    }
+
+                                    val caption = message.caption ?: "none"
+                                    val responseBody = fileInfo.filePath?.let { filePath ->
+                                        withContext(Dispatchers.IO) {
+                                            bot.downloadFile(filePath).toResult().get()
+                                        }
+                                    } ?: return@run Triple(
+                                        message.chat,
+                                        sender,
+                                        RoomMessageEventContent.TextBased.Notice(
+                                            body = "Could not download photo. Caption was: $caption",
+                                        ),
+                                    )
+
+                                    val mxcUrl = client.media.upload(
+                                        Media(
+                                            content = responseBody.byteStream().toByteReadChannel(),
+                                            contentLength = responseBody.contentLength(),
+                                            contentType = responseBody.contentType()?.let {
+                                                ContentType(it.type(), it.subtype())
+                                            },
+                                            contentDisposition = ContentDisposition.Attachment.withParameter(
+                                                "filename",
+                                                "image.jpg",
+                                            ),
+                                        ),
+                                    ).getOrThrow().contentUri
+
+                                    println(message.caption)
+
+                                    return@run Triple(
+                                        message.chat,
+                                        sender,
+                                        RoomMessageEventContent.FileBased.Image(
+                                            fileName = "photo.jpg",
+                                            body = message.caption ?: "photo.jpg",
```

Comment:
If you declare a variable, use it
```suggestion
                                            body = caption ?: "photo.jpg",
```